### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by Bishoy-Samwel**
*  Create a new Rails app called 'hello-rails-react'. ✅ 
*  Initialize your project with Git. ✅ 
*  Set up your project with webpacker the react-rails gems ❗(They are not needed)
*  Make sure that your project has a Postgres database set up. ✅ 
*  Create a table for storing your messages and create 5 different greetings. ✅ 
*  Create an API endpoint that selects a random greeting from your table. ✅ 
*  Create a static view that will be the root of your app. ✅ 
*  Create the App component with react-router and render it in your static view. ✅ 
*  Create the Greeting component that will display a greeting.    ✅ 
*  Set it up as a route in your App component. ✅ 
*  Create a store, an action, and a reducer that will connect to the API endpoint to get the random greeting. ✅ 
*  Display the random greeting in your Greeting component. ✅

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Bishoy-Samwel, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The App is working fine. :heavy_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] Currently, you are using the rails default README file. Please update it to make the README professional. It should have a proper title, description, steps on how to clone and run the project locally, Author link, etc ... so that anyone can understand your project and run it. You can take help from the [README provided by microverse](https://github.com/microverseinc/readme-template).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/controllers/index.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 // Import and register all your controllers from the importmap under controllers/*
```
Comment (SajjadAhmad14) [inline app/javascript/controllers/index.js:1]: Please update your README file as it's asked by the previous code reviewer.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Bishoy, 
 ### You have been doing a tremendous job so far :ok_hand:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by dasileker — APPROVED** [review]
## **Hi @Bishoy-Samwel,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**